wagtail/admin/views/generic/models.py

```diff
@@ -58,6 +55,11 @@ def get_error_message(self):
             return None
         return self.error_message
 
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context['action_url'] = self.get_add_url()
+        return context
+
```

Comment:
I am guessing the issue is this line, however I did not want to just remove it, I assume context is needed but not sure why the add url was added to the create view.

```
context['action_url'] = self.get_add_url()
```

Reply:
@lb- Well caught! `action_url` here is used as the POST target for the 'add' form, but wagtailadmin/shared/header.html is picking it up to be used as the action button. `action_url` is really too generic a name for that, and we should probably get in the habit of using `{% include ... only %}` when including the header to stop unrelated variables from leaking into it - at least on these generic views.

Reply:
Addressed in 1fe36d2597dfae558c979150651feff0f949ca03.

Reply:
Nice, I will take another look